rskj-core/src/fuzzTest/java/co/rsk/net/discovery/PacketDecoderFuzzTest.java

```diff
@@ -0,0 +1,121 @@
+        try {
+            decoder.decodeMessage(ctx, neighborsPeerMessage.getPacket(), sender);
+        } catch (Exception e) {}
+        long end = System.currentTimeMillis();
+
+        if (end - start > 250) {
+            System.out.println("It took too much");
+        }
+    }
+
+    @Tag("DiscoveryPacketDecoderFuzzDecode2")
+    @FuzzTest
+    public void testFuzzDecode2(FuzzedDataProvider data) {
+        ChannelHandlerContext ctx = Mockito.mock(ChannelHandlerContext.class);
+        InetSocketAddress sender = new InetSocketAddress(data.consumeString(256), data.consumeInt(0, 65535));
+        PacketDecoder decoder = new PacketDecoder();
+
+        long start = System.currentTimeMillis();
+        byte[] fuzzData = data.consumeBytes(4096);
+        byte[] mdcCheck = keccak256(fuzzData, 0, fuzzData.length);
+        byte[] msg = Arrays.concatenate(mdcCheck, fuzzData);
+        try {
+            decoder.decodeMessage(ctx, msg, sender);
+        } catch (Exception e) {
+            // Messages will fail really most of the times.
+            // We can eventually have acceptable or unacceptable exceptions, but so far
+            // we will ignore all exceptions
+
+        }
+        long end = System.currentTimeMillis();
+        if (end - start > 100) {
+            start = System.currentTimeMillis();
+            try {
+                decoder.decodeMessage(ctx, fuzzData, sender);
+            } catch (Exception e) {}
+            end = System.currentTimeMillis();
+            if (end - start > 100) {
+                System.out.println("FUZZ DATA: " + Hex.toHexString(fuzzData));
+                throw new RuntimeException("It took " + (end - start) + " with input " + Hex.toHexString(fuzzData));
+            }
+        }
+    }
+
+    @Tag("DiscoveryPacketDecoderFuzzInput")
+    @FuzzTest
+    public void testFuzzInput(FuzzedDataProvider data) {
+        ChannelHandlerContext ctx = Mockito.mock(ChannelHandlerContext.class);
+        InetSocketAddress sender = new InetSocketAddress("localhost", 8080);
+        PacketDecoder decoder = new PacketDecoder();
+        //  Sometimes inputs like this perform slow. There's something in the config,
+        //  gc or whatever that makes it but it is not an issue per se
+        //  But if we can understand the nature of it we can improve benchmark testings
+        byte[] input = Hex.decode("4c4c4c4c4c4c4c4c5c4c4c4c4cbababababababababababa4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c4c");
+        for (int i = 0; i < 100; i++) {
+            long start = System.currentTimeMillis();
+            try {
+                decoder.decodeMessage(ctx, input, sender);
+            } catch (Exception e) {}
+            long end = System.currentTimeMillis();
+            // The threshold here needs to be adjusted, it's always throwing
+            //System.out.println("It took " + (end - start));
```

Comment:
I don't personally like this block here, the catch has only commented out code and a variable that's not used (if I'm not mistaken). Also, I don't like the general "Exception" catching, if the timeout is reported via a Runtime exception, then we should change the type here.